packages/terra-graphs/docs/contributing/README.md

```diff
@@ -0,0 +1,136 @@
+
+Before starting development on a Carbon project, please complete the following steps:
+
+1. If contribution involves changes to `requirements`, create your work in the Carbon JIRA project using the appropriate template:
+    - <a href="https://jira3.cerner.com/browse/CARBON-1/">Enhancement</a>
+    - <a href="https://jira3.cerner.com/browse/CARBON-2/">Defect</a>
+2. For contributions that have an issue in the Github queue, you don't have to do anything specific!
+
+### Development
+
+#### Pulling in dependencies
+
+After checking out the project, you will want to run the following command to pull in dependencies needed by the project:
+
+```sh
+npm install
+```
+
+When contributing to Carbon, you would need to run the below command and load the test page. The test page auto reloads whenever you make any changes to the .CSS, .JS files or if you make any changes to the sample .html file
+
+```sh
+npm run dev
+```
+
+Open in browser:
+
+```sh
+http://localhost:9991/
+```
+
+#### Building the project
+
+To build the project, run the build script:
+
+```sh
+npm run build && npm run build:dist
+```
+
+This will generate `dist` and `lib` folders with `core + packaged dependencies` and `core` source codes respectively.
+
+### Code Review
+
+#### Pull Request checklist
+
+-   Follow the [UX Design Standards](https://wiki.ucern.com/display/UserExperience/Standard+Graphs).
+-   Setup a design meeting to discuss overall design of the project.
+-   Add the code changes.
+-   Add unit tests.
+-   Ensure your code is thoroughly unit tested. (Coverage is at `carbon-graphs\.coverage\html\index.html`)
+-   Ensure no unit tests are broken.
+-   Ensure no linting errors are present.
+-   Add examples in appropriate tab within `dev`.
+-   Write documentation in `Markdown`, following the existing documentation pattern.
+-   Include screenshots and animated GIFs in your **pull request** whenever possible.
+-   Follow proper PR title and Commit message [format](#commit-message-format).
+-   End files with a newline.
+
+#### Commit message format
+
+We follow `ESLint` [commit convention](https://eslint.org/docs/1.0.0/developer-guide/contributing#step-2-make-your-changes).
+Your commit message should look something like this:
+
+```text
+<Tag>: <Short description> (fixes #1234)
+```
+
+**Note**:
+1234 is the Carbon [Github issue](https://github.com/cerner/carbon-graphs/issues/237) number.
+
+#### Pull Request
+
+-   Provide the PR with all necessary information to ease code review process in the `Description`, for instance:
+    -   Screenshots of the working prototype.
+    -   Files you want the reviewer to focus on.
+-   Respond to all comments.
+-   Do **not** rebase with main and force push until you reach `ready to merge` status.
+
+### Release
+
+Release is automated, and will be performed when PR is merged to `main` branch.
+
+carbon-graphs will be released following release types based on the commit message tags
+
+|   Tag    | Release |
```

Comment:
Is this chart still accurate given that terra-graphs is not released automatically when a PR is merged to `main`?

Reply:
Yeah, it needs updating but I'll need more information about the release process to update it.

Reply:
Let's make sure to make the necessary updates once we have gone through and done the first release